pkg/conf/dev.go

```diff
@@ -189,21 +190,59 @@ func (d *DevConfig) RemoveConfigVar(key string) error {
 	return nil
 }
 
-// LoadConfigFile reads the contents of the dev config file at the path
-func (d *DevConfig) LoadConfigFile() error {
+// SetEnvVar updates an env var in the dev config file, creating it if necessary.
+func (d *DevConfig) SetEnvVar(key string, value string) error {
 	d.mu.Lock()
 	defer d.mu.Unlock()
-	config, err := NewDevConfigFile(d.Path)
+	d.EnvVars[key] = value
+
+	if err := writeDevConfig(d); err != nil {
+		return err
+	}
+
+	logger.Log("Successfully wrote environment variable %q to dev config file.", key)
+	return nil
+}
+
+// RemoveEnvVar deletes an env var from the dev config file, if it exists.
+func (d *DevConfig) RemoveEnvVar(key string) error {
```

Comment:
nit - match the command name

```suggestion
func (d *DevConfig) DeleteEnvVar(key string) error {
```